pages/guide/08-start.md

```diff
@@ -0,0 +1,422 @@
+
+Sample Hjson for filtering on multiple fields:
+
+```
+{
+  tables: [{
+      name: PlayerStats
+      sql: '''
+      select name, game_type, created_on, highScore from playerStats where highScore > 100
+      '''
+      dbConnectionName: Presto Data Source
+      friendlyName: Player Stats
+      description:
+      '''
+      A long description
+      '''
+      category: Sports
+      cardinality : large
+      readAccess : '(user AND member) OR (admin.user AND NOT guest user)'
+      isFact : true
+      tags: ['Game', 'Player']
+      measures : [
+          {
+          name : highScore
+          type : INTEGER
+          definition: 'MAX({{highScore}})'
+          friendlyName: High Score
+          }
+      ]
+      dimensions : [
+          {
+              name : name
+              type : TEXT
+              definition : '{{name}}'
+              cardinality : large
+          },
+          {
+              name : gameType
+              type : TEXT
+              definition : '{{game_type}}'
+              friendlyName: Game Type
+          },
+          {
+              name : createdOn
+              type : TIME
+              definition : '{{created_on}}'
+              grain:
+              {
+                  type :  DAY
+                  sql :  '''
+                  PARSEDATETIME(FORMATDATETIME({{}}, 'yyyy-MM-dd'), 'yyyy-MM-dd')
+                  '''
+              }
+          }
+      ]
+  }]
+}
+```
+
+For more information on subqueries please check out: [https://elide.io/pages/guide/v5/04-analytics.html#table-properties](https://elide.io/pages/guide/v5/04-analytics.html#table-properties)
+
+### How often do i need to create a new table?
+
+It totally depends on the use case. You can split a single Physical Table into multiple Logical Tables in Yavin with each table limited to certain type of data. On the other side, you can UNION multiple Physical tables to a single Logic Table in Yavin.
+
+### How do I add another Time Grain in my semantic config?
+
+To add another time grain in a date dimension, we can simply do the following:
+
+```
+    Dimensions: [
+      {
+        # A Date Dimension using multiple time grains (year and day)
+        name: dimension2
+        friendlyName: Dimension 2
+        category: DimCategory2
+        type: TIME
+        definition: '{{release_year}}'
+        grains: [
+          {
+             type: YEAR
+          }
+          {
+             type: DAY
+          }
+        ]
+      }
+        ...
+    ]
+
+```
+
+For more information on subqueries please check out: [https://elide.io/pages/guide/v5/04-analytics.html#time-dimensions--time-grains](https://elide.io/pages/guide/v5/04-analytics.html#time-dimensions--time-grains)
+
+### How can i share my semantic config with others?
+
+You can upload your application code to any source code management software like Github, etc. and share with others.
+
+Similar to the one we have for the demo app and data set:
```

Comment:
Since the questions are independent from each other, i just wanted to point out to those files for there reference location and sample ... so if it is ok with you i will keep.